pkg/karmadactl/cmdinit/options/global.go

```diff
@@ -23,4 +23,6 @@ const (
 	UserName = "karmada-admin"
 	// KarmadaKubeConfigName karmada kubeconfig name
 	KarmadaKubeConfigName = "karmada-apiserver.config"
+	// WaitComponentReadyTimeout wait component ready time
+	WaitComponentReadyTimeout = 30
```

Comment:
@jwcesign Some places use numbers directly, some places use waitPodReadyTimeout. Can we make them unified?
I added a global constant to unify the timeout